File: twiml/voice/recording/recording-1/recording-1.6.x.py
from twilio.twiml.voice_response import Recording, VoiceResponse, Say, Start

response = VoiceResponse()
start = Start()
start.recording(
    channels='dual',
    recording_status_callback='https://example.com/your-callback-url'
)
response.append(start)
response.say('The recording has started.')

print(response)

File: twiml/voice/recording/recording-1/recording-1.6.x.php
<?php
require_once './vendor/autoload.php';
use Twilio\TwiML\VoiceResponse;

$response = new VoiceResponse();
$start = $response->start();
$start->recording(['channels' => 'dual',
    'recordingStatusCallback' => 'https://example.com/your-callback-url']);
$response->say('The recording has started.');

echo $response;
